### PR TITLE
setup: initial

### PR DESCRIPTION
**Pull request by tynes**
WIP

The goal is to have any user facing config in `docker-compose.env` and everything that can be hard coded in `docker-compose.yml`, hiding most of the complexity.

There is a script `$ npm run create-key` which will create a key and add it to `docker-compose.env` so that it will be present at runtime. It will log the address and if the key already exists it will log the same address again.

### CODE REVIEW

**Comment by tynes** [comment]
Additional currently unaccounted for env vars:
- `ETH1_ADDRESS_RESOLVER_ADDRESS`
- `ETH1_L1_CROSS_DOMAIN_MESSENGER_ADDRESS`
- `ETH1_NETWORKID`
- `ROLLUP_ADDRESS_MANAGER_OWNER_ADDRESS`
- `ETH1_CHAINID`

File: createKey.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+      const addr = await w.getAddress()
+      console.log(`Fund the address ${addr}`)
+      process.exit(0)
+    }
+    if (line.startsWith(`#${KEY}=`))
+      line = `${KEY}=${key}`;
+    out.push(line)
+  }
+  await fs.writeFile(cfg, out.join('\n'), 'utf-8')
+  console.log(`Fund the address ${address}`)
```
Comment (ben-chain) [inline createKey.js]:
```suggestion
  console.log(`Generated address for fraud proving: ${address}.  WARNING: THIS KEY IS STORED IN PLAINTEXT.  DO NOT SEND REAL MONEY TO THIS ADDRESS`)
```

or something of the sort 😂

File: docker-compose.env [hunk 2]
```diff
@@ -0,0 +1,4 @@
+ETH1_CTC_DEPLOYMENT_HEIGHT=8
```
Comment (karlfloersch) [inline docker-compose.env]: Is this verifier intended to be run against the local integrations repo?

File: docker-compose.yml [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+services:
+  verifier:
+    image: ethereumoptimism/go-ethereum:latest
+    volumes:
+      - verifier:/root/.ethereum:rw
+    env_file:
+      - docker-compose.env
+    environment:
+      - ROLLUP_VERIFIER_ENABLE=true
+      - CHAIN_ID=420
```
Comment (karlfloersch) [inline docker-compose.yml]: If we want to use this for our fraud proof security drill then we should change this to 69

**Review by karlfloersch — APPROVED** [review]
Left a couple comments but overall this looks great!

**Pull request merged**